src/CssToInlineStyles.php

```diff
@@ -24,28 +24,33 @@ public function __construct()
      * Will inline the $css into the given $html
      *
      * Remark: if the html contains <style>-tags those will be used, the rules
-     * in $css will be appended.
+     * in $css will be appended. Media queries if available will be filtered out
+     * and appended to the body in their own <style> tag.
      *
      * @param string $html
      * @param string $css
      *
      * @return string
      */
-    public function convert($html, $css = null)
+    public function convert(string $html, string $css = null)
```

Comment:
must be reverted. The min PHP version is PHP 5.5

Reply:
Shouldn't all parameter types be removed from the code I implemented?

Reply:
class typehints and array typehints are allowed in PHP 5.3+. Only scalar typehints are an issue (but if you have some in other methods, they should indeed also be removed).

Btw, the CI catched it as well.

Reply:
Thanks, no that was the only spot. PHP is not my native language but I'm sure you noticed that ;-)